modules/model/shared/src/main/scala/observe/model/Step.scala

```diff
@@ -32,6 +33,10 @@ object Step {
   extension [A](l: Lens[A, Boolean]) {
     def negate: A => A = l.modify(!_)
```

Comment:
Do we still need `negate` for other things?

Reply:
it is being called bet we'll probablyy remove it when we take the skipmark off